docs/getting-started-guide/master.adoc

```diff
@@ -33,7 +33,8 @@ include::topics/about-rules.adoc[leveloffset=+2]
 include::topics/migration-paths.adoc[leveloffset=+2]
 
 // Prerequisites
-include::topics/jdk-hardware-mac-prerequisites.adoc[leveloffset=+2]
+== Prerequisites
```

Comment:
```suggestion
.Prerequisites
```

Reply:
== is correct. This is a section in the Introduction to the MTA; it's not part of a procedure.

Reply:
OK. Gotcha.